testing/SPEC-CONFORMANCE.md

```diff
@@ -46,11 +46,11 @@ The conformance requirements for EBU-TT Part 3 derive from the specification its
 |R37|3.2.2.1|~~Every document with the same `ebuttp:sequenceIdentifier` shall be uniquely numbered using the `ebuttp:sequenceNumber` attribute.~~|Deprecated. See R107|
 |R38|3.2.2.2|The following additional metadata elements are defined by this specification. If used these elements shall be the first children of the `ebuttm:documentMetadata` element in the order documented below, after any metadata elements defined by EBU-TT Part 1 Tech 3350.|`bdd/features/validation/documentMetadata_elements_order.feature` `(In)valid documentMetadata elements order`|
 |R39|3.2.2.2|Order of metadata elements (if present) in `ebuttm:documentMetadata` is: `ebuttm:originalSourceServiceIdentifier`, `ebuttm:intendedDestinationServiceIdentifier`, `ebuttm:documentFacet`, `ebuttm:trace`. |`bdd/features/validation/documentMetadata_elements_order.feature` `(In)valid documentMetadata elements order`|
-|R40|3.2.2.2|`ebuttm:documentFacet` : Each distinctly identified `facet` that is summarised shall have a separate `documentFacet` element.|NOTE:Semantic validation for Facet planned to R4|
-|R41|3.2.2.2|Documents shall NOT contain more than one `documentFacet` element referring to the same term, where the term is identified by the combination of the element contents and the value of the link attribute.|NOTE: Semantic validation for Facet planned to R4|
-|R42|3.2.2.2.1|~~`ebuttm:trace` `action` attribute Cardinality 1..1~~ Merged with `ebuttm:appliedProcessing`|`bdd/features/validation/trace.feature` `(In)valid trace attributes`|
-|R43|3.2.2.2.1| ~~`ebuttm:trace` `generatedBy` attribute Cardinality 1..1~~ Merged with `ebuttm:appliedProcessing`|`bdd/features/validation/trace.feature` `(In)valid trace attributes`|
-|R44|3.2.2.3| A document that contains a `tt:body` element with no content shall be treated as being active as defined by the semantics described in § 2.3.1, and shall cause no content to be presented while it is active.|NOTE: the scenarios in `bdd/features/timing/elements_active_times.feature` should cover this although there is no scenario with an empty `<body>`. Add this scenario.|
+|R40|3.2.2.2|~~`ebuttm:documentFacet` : Each distinctly identified `facet` that is summarised shall have a separate `documentFacet` element.~~|Deprecated.|
```

Comment:
Think we should not be misleading here on R40 and R41 - the requirement is not exactly deprecated, but moved to Part M. I would still strike it through though because it has been removed from the Live spec.